src/components/custom/zurich/GlobalNav.tsx

```diff
@@ -237,37 +166,27 @@ const StyledHeaderMain = styled.div`
   }
 `;
 
-function DropdownList(props: NavDropdownProps & { parentName: string }) {
-  const { parentName, items, active } = props;
-  return (
-    <StyledDropdown nav inNavbar className={active && 'active'}>
-      <StyledDropdownToggle nav caret>
-        <NavHighlighter className={`highlighter ${active && 'active'}`}>
-          {parentName}
-        </NavHighlighter>
-      </StyledDropdownToggle>
-      <DropdownMenu direction="left">
-        {items &&
-          items.map((child) => (
-            <DropdownItem key={child.id}>
-              <NavLink>
-                <Link href={child.urlPath}>
-                  <NavHighlighter className="highlighter">{child.name}</NavHighlighter>
-                </Link>
-              </NavLink>
-            </DropdownItem>
-          ))}
-      </DropdownMenu>
-    </StyledDropdown>
-  );
-}
-
 function GlobalNav(props: GlobalNavProps) {
-  const { t } = useTranslation();
+  const { t, i18n } = useTranslation();
   const site = useSite();
   const theme = useTheme();
   const [isOpen, toggleOpen] = useState(false);
-  const { siteTitle, ownerName, navItems, sticky } = props;
+  const { siteTitle, ownerName, navItems } = props;
+
+  const watchLinkTitle = site.watchLink
+    ? t(`watchLink.${site.instanceId}.${i18n.language}`, {
+        defaultValue:
+          typeof site.watchLink.title === 'string'
+            ? site.watchLink.title
+            : site.watchLink.title[i18n.language] || Object.values(site.watchLink.title)[0],
+      })
+    : null;
+
+  const watchLinkUrl = site.watchLink
+    ? typeof site.watchLink.url === 'string'
+      ? site.watchLink.url
+      : site.watchLink.url[i18n.language] || Object.values(site.watchLink.url)[0]
```

Comment:
Nitpick: I see this is taken from `common/GlobalNav`, but we could move this to a dedicated function